src/main/java/com/couchbase/lite/support/CouchbaseLiteHttpClientFactory.java

```diff
@@ -32,7 +32,7 @@
 import okhttp3.OkHttpClient;
 
 public class CouchbaseLiteHttpClientFactory implements HttpClientFactory {
-
+    private OkHttpClient client;
```

Comment:
I created the ticket for refactoring HttpClientFactory.
https://github.com/couchbase/couchbase-lite-java-core/issues/1292